### DISCUSSION TITLE
Supporting a `default` param in `MorphAnalysis.get`

### DISCUSSION THREAD

**Discussion by bryant1410** [explosion/spaCy#12522] [New Features & Project Ideas]
I'm proposing a small change. What about adding a `default` param in `MorphAnalysis.get`, as in [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get)? It'd be helpful to access morphological keys that are often not present.

**Comment by adrianeboyd** [comment]
Thanks, this is a very reasonable suggestion. You are welcome to open a PR if you'd like, or we can add it to our internal to-do list.

**Reply by bryant1410** [reply]
Awesome! Great to hear! I'm not gonna open a PR cause I'm not used to Cython and I feel it'd take me a while to do it properly.

**Reply by adrianeboyd** [reply]
Sure, no problem, please see: #12545
